method for matching

```diff
@@ -0,0 +1,44 @@
+In this paper Thompson argue that Jaffe constructed three patents samples: a set of originating patents, a set of citing patents which reference one of the originated patents and a set of control patents matched to each citing patent. 
+Each control patent shared the same technology class it matched citing patent. The experiment compared the probabilities that the citing patent and its matched control patent, were filed by inventors living in the same geographic location as the originating patent.
+Thompson declares that, controlling for unobservable using matching methods is invariably a dangerous exercise because one can rarely be confident that the control are doing their job. 
+Imperfect matching may simply introduce noise and a corresponding loss of efficiency, but these are not the applications where matching is critical. 
+This next part is to check and defined which method are we using. 
+
+Thompson critics on JTHs method is that:
+1) JTH control patents were selected using broad, three-digit technological classification codes of the U.S. Patent and Trademark Office (USPTO), a level of aggregation that surpasses considerable within-class heterogeneity. 
+This means that, if I match the citing patent with the originating patent using the three digits class, it is also likely to fall in the same subclass. 
+In contrast, the control patent is likely to be drawn from a different subclass, and it will fail to control for pre-existing geographic patterns of production. 
+2) Patents typically contain many distinct claims, to each of which a technological classification is assigned. 
+The particular claim in a citing patent that can be associated with a citation to prior art may be quite distinct from the claim that generated the corresponding control patent. 
+This doesn’t ensure the existence of any industrial link between the originating and control patent.
+
+Thompson claims that these two features of the control selection process means there is no guarantee that the control patent has any industrial similarity either to the citing or to the originating patent. 
+
+Thompson’s method
+1)Construct 2 control groups
+2)First group, for each citing patent he paired it with a control patent having a similar application date and that matched the primary classification of its paired citing patent at the three-digit level. 
+3)Second group, selected control patents that matched the primary classification of its paired citing patent at the level of the subclass. 
+
+Thompson’s criteria.
+1)He select control patents using the technology subclass, a much finer level of disaggregation than the three-digit classification scheme. 
+Instead of using for example class 231, he uses 231.3
+
+2)It focus on the originating-citing-control triads in which all three patents have at least one subclass in common. 
+
+
+JTH selected controls in which the primary class of the citing patent matched the primary class of its control patent. 
```

Comment:
Are we using primary class?

Reply:
Yes, we are using primary class. 
Taken from the NATIONAL BUREAU OF ECONOMIC RESEARCH. 
nclass: US 3-digit current classification (CCL). The variables designated "CCL" are based on the USPTO classification system as of 2008. This means that all the patents on this file will have a consistent classification applied if you use the ccl variables. (I put the excel file with the different classes in the admin folder, called "classification_06 (1)")
The USPTO has developed over the years a highly elaborate classification system for the technologies to which the patented inventions belong, consisting of about 400 main (3-digit) patent classes,13 and over 120,000 patent subclasses. This 400 classes are aggregated into 36 two-digit technological sub-categories, and these in turn are further aggregated into 6 main categories.
http://www.nber.org/papers/w8498.pdf

Also, we are using this file pat76_06_ipc.dta
Which only carries nclass as the US 3-digit current classification
https://sites.google.com/site/patentdataproject/Home/downloads/patn-data-description

Reply:
there's no hurry, but will you please at some point integrate these notes into the original document, along with what you told me today about the other data set that has the sub class?